src/chef-server-ctl/habitat/config/pedant_config.rb

```diff
@@ -199,3 +199,6 @@
 reindex_endpoint "https://127.0.0.1"
 
 chef_pgsql_collector true
+
```

Comment:
I start to wonder if we shouldn't do some rendering of this config based on the state of the server config since there are a few places where a mismatch will break things. (Not for this PR)